fem/post/create_res_sim.py

```diff
@@ -199,18 +210,62 @@ def extract_image_plane(snic, axes, ele_pos):
       ele_pos: elevation position for plane of interest
 
     Returns:
-      image_plane (node IDs)
+        image_plane (node IDs)
 
     """
     import numpy as np
-
-    ele0 = np.min(np.where(axes[0] >= ele_pos))
-    image_plane = np.squeeze(snic['id'][ele0, :, :]).astype(int)
-
+    plane0 = np.min(np.where(axes[direction]>=plane_pos))
+
+    if direction == 0:
+        image_plane = np.squeeze(snic['id'][plane0,:,:]).astype(int)
```

Comment:
Is the only difference in each `elif` block the definition of `image_plane`?  If so, then all of those `print()` statements can be reduced to something outside that conditional logic (i.e., not repeated).  Also, instead of `print()` statements, the package uses the `logging` package; this will both display output to the terminal *and* capture it in a log file so that you can go back and audit what was done.  Check out replacing these prints with `logging.info()` statements.

Reply:
sorry these print statements were when i thought i was writing test cases for the other plane cases and wanted to know for example where image_plane[1][1] was for the coronal plane-- i can just blast them away